### PR TITLE
Feature/playlist

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
# ❓What is the change ❓
- Users can add videos to playlist
- User can create new playlists
- Users can delete playlists.
# What is it ❓
 - [ ] bug🐛
 - [x] feature 👩‍💻
 - [ ] fix 🔧
*To access these features user must be logged in*
Yes, I have not added any toast to notify if user has created playlist or not,
Will raise a PR in the future for that feature.

### CODE REVIEW

File: src/pages/public/Home/Home.jsx [hunk 1]
```diff
@@ -4,11 +4,14 @@ import Navbar from '../../../components/Navbar/Navbar';
...
 import { ToastContainer } from 'react-toastify';
-
+import { useModal } from '../../../context/modal/modalContext';
 import './Home.css';
+import Modal from './../../../components/Modal/Modal';
 
 function Home() {
   const { videoState } = useVideo();
+  const { modalContent } = useModal();
+  console.log(modalContent);
```
Comment (AnkurChunekar) [inline src/pages/public/Home/Home.jsx]: Remove console logs if not needed

File: src/service/getPlaylistItems.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+const getPlaylistItems = async (playlistId, token, setVideos) => {
+  const response = await axios({
+    method: 'GET',
+    url: `/api/user/playlists/${playlistId}`,
+    headers: { authorization: token },
+  });
+  console.log(response);
```
Comment (AnkurChunekar) [inline src/service/getPlaylistItems.js]: This one too

**Review by AnkurChunekar — APPROVED** [review]
- Rest everything looks good.
- playlist functionality works as intended.
- Other than console logs code looks clean

File: src/components/Drawer/Drawer.jsx [hunk 3]
```diff
@@ -22,12 +22,30 @@ function Drawer() {
       >
         <DrawerIcons name="History" type={'HISTORY'} size={'25px'} />
       </NavLink>
-      <NavLink to="/liked">
+      <NavLink
+        style={({ isActive }) => {
+          return isActive ? { color: 'var(--primary-color)' } : null;
+        }}
```
Comment (kart-c) [inline src/components/Drawer/Drawer.jsx:28]: since this function is getting repeated, you can extract it out

**Review by kart-c — APPROVED** [review]
Functionality is working fine. Just a few suggestions I have 

- Show a logout button, when a user is logged in (creates confusion)
- If I add two playlists with the same name then it creates a problem 
  - If I add a video to the playlist, it gets added in both playlists (since the name is same)
  - You can avoid this by having a unique id for all playlists created
  - Or you can restrict the user to not use same names for playlists

File: src/components/Drawer/Drawer.jsx (hunk 3 above)
Comment (Kevin-Solomon) [reply src/components/Drawer/Drawer.jsx:28]: True. will make sure to implement these changes.

**Pull request merged**